riva/clients/nmt/riva_nmt_streaming_s2t_client.cc

```diff
@@ -124,7 +129,7 @@ main(int argc, char** argv)
 
   std::signal(SIGINT, signal_handler);
   gflags::ParseCommandLineFlags(&argc, &argv, true);
-
+  emptyFileBeforeRequest();
```

Comment:
do we need this function? at the start of fresh run of client, new file should get created that would overwrite any previous content. We should not need to explicitly empty previously existing file.